clj/resources/static_content/js/module.js

```diff
@@ -529,6 +529,7 @@ jQuery( function() { ( function( $$, $, undefined ) {
         });
 
         $btnRefreshPRS.click(function() {
+            cachedPRSResponse = null;
```

Comment:
I believe that there is also caching/memoization of the PRS responses on the server side.  You may also want to check if it in fact does this and if so remove the memoization.